src/ApigeeSdkControllerFactory.php

```diff
@@ -183,6 +186,24 @@ public function companyBalanceController(CompanyInterface $company): CompanyPrep
     return $this->controllers[__FUNCTION__][$name];
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function appGroupBalanceController(AppGroupInterface $appgroup): AppGroupPrepaidBalanceControllerInterface {
```

Comment:
Changes missing in ApigeeSdkControllerFactoryInterface?

Reply:
Done